app/allocations/middleware/set-body.allocations.js

```diff
@@ -6,6 +6,8 @@ function setBodyAllocations(req, res, next) {
 
   set(req, 'body.allocations', {
     status,
+    sortBy: req.query.sortBy,
+    sortDirection: req.query.sortDirection,
```

Comment:
Think the initial comment here got missed so adding to this line instead. It would be good to destructure these from `req.query` like `status` is so that we don't have two methods for getting values from `req.query`.

Reply:
👍